bash-completion.yaml

```diff
@@ -0,0 +1,61 @@
+package:
+  name: bash-completion
+  version: 2.14.0
+  epoch: 0
+  description: "Programmable completion functions for bash"
+  copyright:
+    - license: GPL-2.0
+  dependencies:
+    runtime:
+      - bash
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/scop/bash-completion
+      expected-commit: 0543d1a28ce3d36741675c7ef6da7c2286288f3e
+      tag: ${{package.version}}
+
+  - runs: autoreconf -fiv
+
+  - uses: autoconf/configure
+    with:
+      opts: |
+        --host=${{host.triplet.gnu}} \
+        --target=${{host.triplet.gnu}} \
+        --prefix=/usr \
+        --bindir=/bin \
+        --sysconfdir=/etc \
+        --without-libidn \
+        --with-ssl=openssl \
+        --disable-nls \
+        --enable-readline \
+        --without-bash-malloc \
+        --with-curses
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+update:
+  enabled: true
+  github:
+    identifier: scop/bash-completion
+    use-tag: true
+    tag-filter: v
```

Comment:
I think `tag-filter` can be removed looking at recent tags https://github.com/scop/bash-completion/tags

Reply:
Good call, `2.` is a better filter here (without it we get some of the old junk tags like `rel20050121`).